### PR TITLE
Add sign in configuration.

### PR DESCRIPTION
**Pull request by julianoviedo1**
In this pr i had also added I18n implementation.

Here it is the sign in view:

image

### CODE REVIEW

File: app/assets/stylesheets/bootstrap/dist/css/bootstrap-grid.css.mapZone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (jpbalarini) [inline app/assets/stylesheets/bootstrap/dist/css/bootstrap-grid.css.mapZone.Identifier]: I'm not sure if all these zonefiles need to be commited or should be added to the gitignore. First time I see these

File: app/assets/stylesheets/bootstrap/scss/_accordion.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  width: 100%;
+  padding: $accordion-button-padding-y $accordion-button-padding-x;
+  @include font-size($font-size-base);
+  color: $accordion-button-color;
+  text-align: left; // Reset button style
+  background-color: $accordion-button-bg;
+  border: 0;
+  @include border-radius(0);
+  overflow-anchor: none;
+  @include transition($accordion-transition);
```
Comment (jpbalarini) [inline app/assets/stylesheets/bootstrap/scss/_accordion.scss:18]: are we overriding all these properties or is this just a copy/paste from the original class?

File: app/views/devise/shared/_links.html.erb [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 <% end %>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to t(".sign_up"), new_registration_path(resource_name), class:'text-muted fs-5 fw-bold'  %><br />
 <% end %>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+  <%= link_to t(".forgot_your_password"), new_password_path(resource_name), class:'text-muted fs-5' %><br />
```
Comment (jpbalarini) [inline app/views/devise/shared/_links.html.erb:10]: Let's be consistent with quotes. Let's always use single quotes for ruby code and double quotes for HTML/CSS/JS

**Review by jpbalarini — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/bootstrap/dist/css/bootstrap-grid.css.mapZone.Identifier (hunk 1 above)
Comment (julianoviedo1) [reply app/assets/stylesheets/bootstrap/dist/css/bootstrap-grid.css.mapZone.Identifier]: Ive erased that files in commits b86c5cf(dist folder) and 7b75ce4

File: app/assets/stylesheets/bootstrap/scss/_accordion.scss (hunk 2 above)
Comment (julianoviedo1) [reply app/assets/stylesheets/bootstrap/scss/_accordion.scss:18]: Its just a copy paste! from the original an only

File: app/views/devise/shared/_links.html.erb (hunk 3 above)
Comment (julianoviedo1) [reply app/views/devise/shared/_links.html.erb:10]: Great, i ve fixed quotes in commit 1d4d654

**Review by jpbalarini — APPROVED** [review]
Looking good!

**Pull request merged**